src/misc/utils/formatBalance.tsx

```diff
@@ -0,0 +1,148 @@
+
+const divmod = (dividend: number, divider: number) => {
+    let quotient = Math.floor(dividend / divider);
+    let remainder = dividend % divider;
+    return [quotient, remainder];
+}
+
+// found on World Wide Web.
+// It is just to format very small number from scientific notation to eg. 0.00000001
```

Comment:
https://github.com/galacticcouncil/HydraDX-app/blob/d2d5764839988dd9ec0dd08b7867137f52de5f91/src/components/common/BalanceInput.vue#L17

wdyt about using bignumber like here and just trimming decimals for display?